integreat_cms/cms/constants/roles.py

```diff
@@ -101,6 +101,7 @@
     "delete_mediafile",
     "delete_organization",
     "grant_page_permissions",
+    "manage_translations",
```

Comment:
Setting new permissions also require a database migration to apply them on the existing production systems.

Reply:
I'm not really clear on this. Do I need to add a migration like [this one](https://github.com/digitalfabrik/integreat-cms/blob/f973afc2eb7813ea96dd434c235c69cb3f4afd50/integreat_cms/cms/migrations/0005_grant_imprint_deletion_permission.py)? Attempting this results in an `Permission matching query does not exist.` error during migration.

Reply:
Yes, when you change permissions which are created in the same migration, you have to emit the post migrate signal in order for the auth app to pick up the model changes and create the permission object, e.g. like here:

https://github.com/digitalfabrik/integreat-cms/blob/f973afc2eb7813ea96dd434c235c69cb3f4afd50/integreat_cms/cms/migrations/0044_alter_user_options.py#L22-L23

Reply:
Or wait, where is this permission assigned? If it doesn't exist yet, you have to add it to one of the models, probably the user model?

Reply:
> probably the user model

Thanks, that did it!